### DISCUSSION TITLE
v2.0.2-b7无法使用显示断开

### DISCUSSION THREAD

**Discussion by x51d6g4** [cangzhang/ChampR#290] [General]
打开后无法正常使用，杀毒软件什么的均已关闭也无法使用
后台开着steamcommunity_302本地反代用于访问github
![QQ截图20230821122134](https://github.com/cangzhang/champ-r/assets/142769644/e1025964-a0c4-44e4-8465-ea0cb62f23cc)
[log.2023-08-21.log](https://github.com/cangzhang/champ-r/files/12391820/log.2023-08-21.log)

**Comment by cangzhang** [comment]
看起来是访问不到服务器。试试下面的网址可以在浏览器打开吗？
https://ql-rs.lbj.moe/api/data-dragon/champions

**Comment by x51d6g4** [comment]
https://ql-rs.lbj.moe/api/data-dragon/champions
网址打不开

**Reply by cangzhang** [reply]
那看起来是你本地网络环境问题了。加个代理试试

**Comment by x51d6g4** [comment]
> 那看起来是你本地网络环境问题了。加个代理试试
没有代理 暂时不能使用啦。。。。
